.github/workflows/set_community_label.yml

```diff
@@ -5,7 +5,7 @@ on:
     types: [opened]
 
 permissions:
-  contents: read
+  issues: write
```

Comment:
We should probably add a `permissions:` block to the `Set Community Label If Needed` step rather than at the top-level for this action. I would suggest reverting this change and adding
```
permissions:
    issues: write
``` 
at line 35